pyface/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 
 __requires__ = [
-    'importlib-metadata; python_version<"3.8"',
+    'importlib-metadata>=3.6.0; python_version<"3.8"',
```

Comment:
the python version requirement also changes here - the external `importlib-metadata` is necessary for all versions < 3.10. See compatibility note in https://docs.python.org/3.10/library/importlib.metadata.html#entry-points 

```suggestion
    'importlib-metadata>=3.6.0; python_version<"3.10"',
```